src/validator.test.ts

```diff
@@ -0,0 +1,27 @@
+import {expect, test} from "@jest/globals";
+
+import {CONTRACT_VERSION} from "./contract/constants";
+import {SupportedNetwork} from "./util/commonTypes";
+import {getValidatorAppID, VALIDATOR_APP_ID} from "./validator";
+
+describe("`getValidatorAppID`", () => {
+  test("throws an error when one of the args is invalid", () => {
+    expect(() =>
+      getValidatorAppID("unknown" as any, CONTRACT_VERSION.V1_1)
+    ).toThrowError();
+    expect(() => getValidatorAppID("mainnet", "unknown" as any)).toThrowError();
+  });
+
+  test("returns the correct validator app id", () => {
```

Comment:
I'm not very confident that this is a useful test 😄 Open for comments

Reply:
We simply access a field of an object and return it. We will use the same object even if we have that test so I think we do not need that test.

Reply:
I think it doesn't hurt and offers some safety